docs/model-inputs.md

```diff
@@ -0,0 +1,289 @@
+- Initial implementation assumes that LITTER_WATER is not on. This might be revisited at a later date.
+
+
+### Fertilization
+
+| parameter |  col  | req?  | description |
+| --------- | :---: | :---: | ----------- |
+| org-N     |   5   |   Y   | g N / m2    |
+| org-C     |   6   |   Y   | g C / m2    |
+| min-N     |   7   |   Y   | g N / m2    |
+<!--(NH4+NO3 in one pool model; NH4 in two pool model)
+| min-N2 |   8   | Y*          | g N / m2 (*not unused in one pool model, NO3 in two pool model) | 
+-->
+
+  - Model representation: increases size of mineral N and litter C and N. Urea-N is assumed to be mineral N.
+<!-- or NH4 in two pool model ... common assumption (e.g. DayCent) unless urease inhibitors are represented.-->
+  - The code that generates `events.in` will handle conversion from fertilizer amount and type to mass of N and C allocated to different pools. In PEcAn this is done by the `PEcAn.SIPNET::write.configs.SIPNET()` function.
+
+### Tillage
+
+| parameter                        |  col  | req?  | description         |
+| -------------------------------- | :---: | :---: | ------------------- |
+| tillageEff $(f_{\textrm{till}})$ |   5   |   Y   | Adjustment to $R_H$ |
+
+- Model representation:
+  - Transient increase in decomposition rate by $f_{\text{,tillage}}$ that exponentially decays over time.
+  - Multiple tillage events are additive.
+
+### Planting
+
+| parameter     |  col  | req?  | description                                  |
+| ------------- | :---: | :---: | -------------------------------------------- |
+| leaf-C        |   5   |   Y   | C added to leaf pool (g C / m2)              |
+| wood-C        |   6   |   Y   | C added to above-ground wood pool (g C / m2) |
+| fine-root-C   |   7   |   Y   | C added to fine root pool (g C / m2)         |
+| coarse-root-C |   8   |   Y   | C added to coarse root pool (g C / m2)       |
+
+- Model representation:
+  - Date of event is the date of emergence, not the date of actual planting 
+  - Increases size of carbon pools by the amount of each respective parameter
+  - $N$ pools are calculated from $CN$ stoichiometric ratios.
+- notes: PFT (crop type) is not an input parameter for a planting event because SIPNET only represents a single PFT.
+
+### Harvest
+
+| parameter                                                  |  col  | req?  | description           |
+| ---------------------------------------------------------- | :---: | :---: | --------------------- |
+| fraction of aboveground biomass removed                    |   5   |   Y   |                       |
+| fraction of belowground biomass removed                    |   6   |   N   | default = 0           |
+| fraction of aboveground biomass transferred to litter pool |   7   |   N   | default = 1 - removed |
+| fraction of belowground biomass transferred to litter pool |   8   |   N   | default = 1 - removed |
```

Comment:
If I'm interpreting the current code right these are all required at the moment. Is there a plan to change this?

Reply:
@infotroph good catch, and yes it appears that these are all required in the current implementation. These defaults make sense for annuals but not perennials, so I think it should stay this way for simplicity.


```suggestion
| fraction of belowground biomass removed                    |   6   |   Y   | |
| fraction of aboveground biomass transferred to litter pool |   7   |   Y   | |
| fraction of belowground biomass transferred to litter pool |   8   |   Y   | |
```

@Alomir leaving this for you to make sure we are on the same page about the updated requirements.

Reply:
FYI the requirement is enforced by https://github.com/PecanProject/sipnet/blob/85a136bf6078b2b2943d92b06a577fde15cc0844/src/sipnet/events.c#L47-L49